src/main/resources/org/sagebionetworks/web/client/widget/entity/EntityPageTopViewImpl.ui.xml

```diff
@@ -8,28 +8,28 @@
 >
   <g:HTMLPanel>
     <!--Project level information -->
-    <bh:Div
```

Comment:
Pretty much all of the XML changes are 
1. Rearranging structure and modifying styles to get consistent margin/padding
2. Removing fields that have been replaced by more action menu actions